src/site/templates/rightcolumn.gsp

```diff
@@ -4,12 +4,16 @@
     def sourceFileName = content?.uri?.replaceAll("[.]"+content.outfilesuffix[1..-1],content.docfilesuffix)
     def subject = java.net.URLEncoder.encode("Docs: Feedback for '${content?.title}'", "UTF-8")
 %>
+<% if (config.site_gitRepoUrl != null) {%>
         <a href="${config.site_gitRepoUrl}/${sourceFileName}"
            target="_blank"><i class="fa fa-edit fa-fw"></i> Improve this doc</a>
+<% } %>
         <!--a href="https://github.com/google/docsy-example/new/master/content/en/docs/_index.md?filename=change-me.md&amp;value=---%0Atitle%3A+%22Long+Page+Title%22%0AlinkTitle%3A+%22Short+Nav+Title%22%0Aweight%3A+100%0Adescription%3A+%3E-%0A+++++Page+description+for+heading+and+indexes.%0A---%0A%0A%23%23+Heading%0A%0AEdit+this+template+to+create+your+new+page.%0A%0A%2A+Give+it+a+good+name%2C+ending+in+%60.md%60+-+e.g.+%60getting-started.md%60%0A%2A+Edit+the+%22front+matter%22+section+at+the+top+of+the+page+%28weight+controls+how+its+ordered+amongst+other+pages+in+the+same+directory%3B+lowest+number+first%29.%0A%2A+Add+a+good+commit+message+at+the+bottom+of+the+page+%28%3C80+characters%3B+use+the+extended+description+field+for+more+detail%29.%0A%2A+Create+a+new+branch+so+you+can+preview+your+new+file+and+request+a+review+via+Pull+Request.%0A"

```

Comment:
Do we need to keep the commented lines?

If no -> I would remove them.

If yes -> I would turn them to `gsp` comment so that the HTML comments are not present in the generated pages.

Reply:
good question. I guess we can remove them.

... and I think we should not test for `null` but use the groovy-truth (https://blog.mrhaki.com/2009/08/groovy-goodness-tell-groovy-truth.html)

An empty String will be false, so

<% if (config.site_gitRepoUrl) {%>

will test if the string is set.

Reply:
Thank you for this groovy hint.

I wanted exactly is "not null and not empty" but I was not aware how to express this. The *groovy-truth* is exactly what we need here.